c2corg_ui/static/js/auth/auth.js

```diff
@@ -186,7 +186,11 @@ app.AuthController.prototype.successLogin_ = function(remember, response) {
  */
 app.AuthController.prototype.register = function() {
   var alerts = this.alerts_;
-  this.api_.register(this.scope_['register']).then(function() {
+  var lang = window.localStorage.getItem('interface_lang') || 'fr';
```

Comment:
Isn't ``interface_lang`` stored in a cookie?
See https://github.com/c2corg/v6_ui/search?utf8=%E2%9C%93&q=interface_lang